deploy/ansible/roles/iroha-k8s/files/k8s-iroha.yaml

```diff
@@ -29,6 +29,26 @@ spec:
         app: iroha
     spec:
       terminationGracePeriodSeconds: 7
+      initContainers:
+      - name: copy-data
+        image: busybox
+        command: ['sh', '-c', 'cp -v /opt/secrets/$PODNAME/node* /opt/iroha_data && cp -v /opt/iroha_config/* /opt/iroha_data && tail /opt/iroha_data/*']
+        env:
+          - name: PODNAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+        volumeMounts:
+        - name: iroha-secret0
+          mountPath: /opt/secrets/iroha-0
+        - name: iroha-secret1
+          mountPath: /opt/secrets/iroha-1
+        - name: iroha-secret2
+          mountPath: /opt/secrets/iroha-2
```

Comment:
I'm not really aware of the k8s yaml syntax, but it seems there's only 3 nodes. If so, would be nice to extend it to 4 (so BFT consensus feature isn't useless)

Reply:
You're right I was only using three nodes in my generated statefulset, I've switched to four